src/util/tr.js

```diff
@@ -635,7 +625,27 @@ const _tr = (msg, {
       }
     } else {
       window.clevertap.popupCurrentWzrkId = targetingMsgJson.wzrk_id
-      renderFooterNotification(targetingMsgJson)
+      if (displayObject.deliveryTrigger) {
+        if (displayObject.deliveryTrigger.inactive) {
+          triggerByInactivity(targetingMsgJson)
+        }
+        if (displayObject.deliveryTrigger.scroll) {
+          triggerByScroll(targetingMsgJson)
+        }
+        if (displayObject.deliveryTrigger.isExitIntent) {
+          exitintentObj = targetingMsgJson
+          window.document.body.onmouseleave = showExitIntent
+        }
+        // delay
+        if ((displayObject.delay != null && displayObject.delay > 0) || (displayObject.deliveryTrigger.deliveryDelayed != null && displayObject.deliveryTrigger.deliveryDelayed > 0)) {
+          const delay = displayObject.deliveryTrigger.deliveryDelayed ? displayObject.deliveryTrigger.deliveryDelayed : displayObject.delay
+          setTimeout(() => {
+            renderFooterNotification(targetingMsgJson)
+          }, delay * 1000)
```

Comment:
This can be optimised to something like 
const delay = displayObject.deliveryTrigger.deliveryDelayed || displayObject.delay;
if (delay > 0) {
  setTimeout(() => renderFooterNotification(targetingMsgJson), delay * 1000);
}

Reply:
Yes let me update it!

Reply:
Done